docs/image/skymaps.rst

```diff
@@ -63,3 +63,72 @@ The sky map can be easily displayed with an image viewer, by calling ``skymap.sh
         counts = SkyMap.read(filename)
         counts.name = 'Counts'
         counts.show()
+
+
+Working with the cutout and paste method
+----------------------------------------
+
+The `~gammapy.image.SkyMap` class offers `.paste()` and `.cutout()`
+methods, that can be used to cut out smaller parts of a sky map.
+Here we cut out a 2 deg x 2 deg patch out of an example image:
```

Comment:
Is it 2 x 2 deg or 1 x 1 deg?
Below you pass `size = Quantity([1, 1], 'deg')` ... so it's 1 x 1 deg, no?